coral/src/app/main.module.css

```diff
@@ -1,17 +1,3 @@
-/*@TODO use aiven-ds based values or global utility class*/
-:global(.visually-hidden) {
-  position: absolute !important;
-  width: 1px !important;
-  height: 1px !important;
-  padding: 0 !important;
-  margin: -1px !important;
-  overflow: hidden !important;
-  clip: rect(0, 0, 0, 0) !important;
-  white-space: nowrap !important;
-  border: 0 !important;
-  font-size: 18px;
-}
-
 /* @TODO check design system */
```

Comment:
Should we also have a constant in `accessibility` for the color in the `body` rule underneath?

```
body {
  margin: 0;
  /*@TODO check with design, it is supposed*/
  /*  to be grey-60 but that does not meet*/
  /*  accessibility standards*/
  /*represents grey-70*/
  color: #4a4b57;
  -webkit-text-size-adjust: 100%;
}
```